gui/pages/Content/Models/ModelInfo.js

```diff
@@ -0,0 +1,32 @@
+import React, {useState, useEffect} from "react";
+import Image from "next/image";
+
+export default function ModelInfo(modelDetails){
+    const [modelData, setModelData] = useState(modelDetails.modelDetails)
```

Comment:
what if modelDetails is empty or null, shouldn't it be useState(modelDetails?.modelDetails)

Reply:
done